etherhoused-log

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exec papertrail --min-time "24 hours ago" "program:app/web.1 AND (\"turn_off ${1}\" OR \"turn_on ${1}\")"
```

Comment:
Hmm. Actually I wonder if it wouldn't be better to just match on the text "house_id:${1}" here, and adjust the daemon to log everything related to a house (except auth failures I suppose) with the house ID in that format. That'd allow me to see how often the status and MAC queries were executed too, which would then tell me how often my system is rebooting.